src/app/app.module.ts

```diff
@@ -10,9 +10,10 @@ import { AppRoutingModule } from './app-routing.module';
 import { HomeComponent } from './components/home/home.component';
 import { PageNotFoundComponent } from './components/layout/page-not-found/page-not-found.component';
 import { HeaderComponent } from './components/layout/header/header.component';
-import { SearchUserComponent } from './components/search-user/search-user.component';
+import { UserSearchComponent } from './components/user-search/user-search.component';
 import { ShowUserComponent } from './components/show-user/show-user.component';
-import { DataStoreService } from './services/data-store.service';
+import { DataStore } from './services/data.store';
+import { SearchbarComponent } from './components/searchbar/searchbar.component';
```

Comment:
Follow proper camle casing with names. `SearchbarComponent` should be `SearchBarComponent` or `SearchBoxComponent`.